metrics/report/report_dockerfile/scaling.R

```diff
@@ -55,26 +55,29 @@ for (currentdir in resultdirs) {
 			fdata=fdata[[shortname]]
 
 			testname=datasetname
-
+			
```

Comment:
`git diff` is showing me trailing whitespace on two lines here I think.

Reply:
haven't found a great vim plugin for R yet, so I keep missing these. I have tabs highlighted, but so many tabs, I keep missing the hanging ones.